server/gql/models/products/index.js

```diff
@@ -3,7 +3,7 @@ import { createConnection } from 'graphql-sequelize';
 import { getNode } from '@gql/node';
 import { SupplierConnection } from '../suppliers';
 import { storeQueries } from '../stores';
-import { timestamps } from '../timestamps';
+import { timestamps } from '../../fields/timestamps';
```

Comment:
let's write an alias for fields. In the webpack configuration you'll see that we have the alias key.

I want it such that we can import using the following statement

```
import { timestamps } from '@gqlFields/timestamps';
```

Reply:
Ok, I understand.